packages/editor/src/components/preview-dropdown/index.js

```diff
@@ -170,6 +170,10 @@ export default function PreviewDropdown( { forceIsAutosaveable, disabled } ) {
 										: 'post-only';
 									setRenderingMode( newRenderingMode );
 									setDefaultRenderingMode( newRenderingMode );
+									// Reset zoom out if show template is being toggled off
+									if ( newRenderingMode === 'post-only' ) {
+										resetZoomLevel();
+									}
```

Comment:
```suggestion
									resetZoomLevel();
```

I don't think there's any problem with disabling zoom out whether the template is being toggled off or not. What do you think?

Reply:
Makes sense! I've removed the conditional and now resetZoomLevel() runs regardless of rendering mode.